clippy_lints/src/returns.rs

```diff
@@ -289,3 +291,17 @@ fn last_statement_borrows<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx Expr<'tcx>)
     })
     .is_some()
 }
+
+// Go backwards while encountering whitespace and extend the given Span to that point.
+#[expect(clippy::cast_possible_truncation)]
+fn extend_span_to_previous_non_ws(cx: &LateContext<'_>, sp: Span) -> Span {
+    if let Ok(prev_source) = cx.sess().source_map().span_to_prev_source(sp) {
+        let ws = [' ', '\t', '\n'];
+        if let Some(non_ws_pos) = prev_source.rfind(|c| !ws.contains(&c)) {
+            let len = prev_source.len() - non_ws_pos - 1;
+            return sp.with_lo(BytePos(sp.lo().0 - len as u32));
```

Comment:
```suggestion
            return sp.with_lo(sp.lo() - BytePos::from_usize(len));
```

Reply:
Thank you for your quick response.
I resolved it in https://github.com/rust-lang/rust-clippy/pull/9967/commits/39d0477080de2c458cdbeb4a0a44d267d58af521.